showcase/linked-cohort-data.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Linked Cohort Data
+title: Linked pathogen and host data across archives - a multi-omics, SARS-CoV-2 cohort case study
 contributors: [Gabriele Rinck, Zahra Waheed] 
-description: Here we describe the first example of linking and sharing a cohort data set via the EMBL-EBI infrastructure. 
-affiliations: [ELIXIR Europe]
+description: Here we describe the first example of linking and sharing a COVID-19, multi-omics cohort data set via the EMBL-EBI infrastructure. 
+affiliations: [EMBL-EBI]
```

Comment:
don't forget to add information about this affiliation to the _data/affiliations.yml file!

Reply:
Ah thanks, done!